### PR TITLE
Device status endpoint & announcement feature

### PR DESCRIPTION
**Pull request by garhul**
Upon receiving and processing a command (fx, cmd, etc) the device notifies any listener via mosquitto on the topic composed of `topic_base` + `/state`, when a command is issued via http the response of the http command is the new state of the strip.

### CODE REVIEW

File: src/Strip.h [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
+//TODO :: change this to not need float conversion
 #define REL_UNIT_BYTE 0.0039f
-#define REL_UNIT_SIZE 1.0f /  STRIP_SIZE
+
 // #define BENCHMARK
 #define MAX_LENGTH 300 //TODO:: temporal solution for dinamic array implementation
 
 #define CMD_FX "fx"
 #define CMD_OFF "off"
-#define CMD_SETHSL "setHsl"
```
Comment (sfabrizio) [inline src/Strip.h:11]: why rename this? `setHsl` is more descriptive.. `set` might be whatever.. no even a color.

File: src/main.cpp [hunk 2]
```diff
@@ -23,13 +21,14 @@ void setup(void) {
 
   Network::init();
   if (Network::getMode() == Network::MODES::ST) {
-    if (Settings::use_mqtt && Mosquitto::init(messageHandler)) {
+    if (Settings::use_mqtt && Mosquitto::init(strip)) {
```
Comment (sfabrizio) [inline src/main.cpp:24]: I think we should remove this `use_mqtt` flag.. for what I'm seeing aurora can't do much without it.

**Review by sfabrizio — CHANGES_REQUESTED** [review]
Looks good to me. I didn't get the cmd rename only.

**Review by sfabrizio — APPROVED** [review]

File: src/main.cpp (hunk 2 above)
Comment (garhul) [reply src/main.cpp:24]: There's many use cases where we don't use mqtt at all, for instance the web UI doesn't require it, this allows for optional usage of mqtt

File: src/Strip.h (hunk 1 above)
Comment (garhul) [reply src/Strip.h:11]: this is indeed strange, I didn't remove it myself, it's probably a bad rebase

**Pull request merged**